lib/deaf_grandma.rb

```diff
@@ -17,11 +17,21 @@ def run!
     end
   end
 
-
+  
   def speak(input)
-
     #Implement your code here <<<<<<<<<
-
+    if input == input.upcase
```

Comment:
Sorry, the '!' was a typo. Sure, I will make the changes. Thank you!

On Thu, Jan 14, 2016 at 1:18 PM, Joe Reich <notifications@github.com> wrote:

> In lib/deaf_grandma.rb
> <https://github.com/paircolumbus/MoreDrillsWithRSpec/pull/39#discussion_r49762679>
> :
>
> >      #Implement your code here <<<<<<<<<
> > -
> > +    if input == input.upcase
>
> rubocop and @paradime <https://github.com/paradime> recommend using a
> 'guard clause' instead of wrapping an entire method in an if-statement.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/paircolumbus/MoreDrillsWithRSpec/pull/39/files#r49762679>
> .
>